lib/MusicBrainz/Server/Controller/Artist.pm

```diff
@@ -289,8 +306,12 @@ sub show : PathPart('') Chained('load')
             ajaxFilterFormUrl => $c->uri_for_action('/ajax/filter_artist_release_groups_form', { artist_id => $artist->id }),
             artist => $artist,
             filterForm => $c->stash->{filter_form},
+            hasDefault => boolean_to_json($has_default),
+            hasExtra => boolean_to_json($has_extra),
             hasFilter => boolean_to_json($has_filter),
-            includingAllStatuses => $including_all_statuses,
+            hasVA => boolean_to_json($has_va),
+            hasVAExtra => boolean_to_json($has_va_extra),
```

Comment:
I think we shouldn't abbreviate VA here since we don't in `showingVariousArtistsOnly`